app/models/form_application.rb

```diff
@@ -0,0 +1,18 @@
+class FormApplication < ActiveRecord::Base
```

Comment:
Let us merge the draft functionality provided by `FormApplication` into `Application`. But since the added model won't conflict with anything, I will merge it first and we can go from there.